src/internal/modules/charge-information/routes/charge-information.js

```diff
@@ -99,7 +99,9 @@ module.exports = {
       pre: [
         { method: preHandlers.loadDraftChargeInformation, assign: 'draftChargeInformation' },
         { method: preHandlers.loadLicence, assign: 'licence' },
-        { method: preHandlers.loadIsChargeable, assign: 'isChargeable' }
+        { method: preHandlers.loadIsChargeable, assign: 'isChargeable' },
+        { method: preHandlers.loadLicenceVersions, assign: 'licenceVersions' },
```

Comment:
the licenceVersions naming here is misleading because you are returning a single object (the oldest licence version) and not an array.